playbook/app/pb_kits/playbook/pb_text_input/_text_input.tsx

```diff
@@ -101,8 +180,9 @@ const TextInput = (props: TextInputProps, ref: React.LegacyRef<HTMLInputElement>
         id={id}
         key={id}
         name={name}
-        onChange={onChange}
-        placeholder={placeholder}
+        onChange={handleChange}
```

Comment:
I was debating changing this to:

`onChange={onChange || (isMaskedInput ? handleChange : onChange)}`

but, I think this would be the same?

Reply:
Made this change!